orocos.osdeps

```diff
@@ -93,7 +93,10 @@ graphviz:
     macos-brew: graphviz
 
 gccxml:
-    debian,ubuntu: gccxml
+    debian: gccxml
```

Comment:
'default' should be the future-proof entry. Manually list all versions since 12.04 (I know, I know) and then set `default: nonexistent`